src/api/services/PriceTickerService.ts

```diff
@@ -136,12 +181,12 @@ export class PriceTickerService {
      * @param currency
      * @returns {Promise<PriceTicker>}
      */
-    public async needTobeUpdate(priceTicker: PriceTicker): Promise<boolean> {
+    private async needTobeUpdate(priceTicker: PriceTicker): Promise<boolean> {
         const diffMint = await this.checkDiffBtwDate(priceTicker['updatedAt']);
         return (diffMint > process.env.DATA_CHECK_DELAY) ? true : false;
     }
 
-    public async getAndCreateData(currency: string): Promise<any> {
+    private async getAndCreateData(currency: string): Promise<any> {
         const returnData: any = [];
```

Comment:
what is this returning?

Reply:
I have removed that and put single function for update and check